src/server/response.rs

```diff
@@ -0,0 +1,22 @@
+// Copyright (c) 2021 Western Digital
+// Licensed under the Apache License, Version 2.0, see LICENSE for details.
+// SPDX-License-Identifier: Apache-2.0
+
+//! The ability for an RoT to respond to requests.
+
+use crate::mem::Arena;
+use crate::net;
+use crate::server::Error;
+
+/// The ability for an RoT to respond to requests.
+pub trait Response {
```

Comment:
Mumble maybe this should be `Respond`? I worry about this getting mixed up with the response types from the protocol module.